docs/snapshot-controller.md

```diff
@@ -10,13 +10,11 @@ flowchart TD
 
   predicate((PREDICATE: <br>Snapshot got created OR <br> changed to Finished OR <br> re-run label added AND <br> it's not restored from backup))
 
-  %%%%%%%%%%%%%%%%%%%%%%% Drawing EnsureStaticIntegrationPipelineRunsExist() function
+  %%%%%%%%%%%%%%%%%%%%%%% Drawing EnsureIntegrationPipelineRunsExist() function
 
   %% Node definitions
   ensure1(Process further if: Snapshot testing <br>is not finished yet)
   are_there_any_ITS{"Are there any <br>IntegrationTestScenario <br>present for the given <br>Application?"}
-  does_ITS_has_env_defined{Does the <br>IntegrationTestScenario <br>has any environment <br>defined in it?}
-  skip_creating_test_PLR(Skip creating Test PLR for this ITS,<br> as it will be created by binding controller)
   create_new_test_PLR(<b>Create a new Test PipelineRun</b> for each <br>of the above ITS, if it doesn't exists already)
   mark_snapshot_InProgress(<b>Mark</b> Snapshot's Integration-testing <br>status as 'InProgress')
```

Comment:
I realize the `<b> </b>` is being used incorrectly in snapshot controller diagram https://github.com/redhat-appstudio/integration-service/blob/main/docs/snapshot-controller.md, but it should not be included in this PR. We may fix it after merging this PR.